test/uk/gov/hmrc/teamsandrepositories/services/PersistingServiceSpec.scala

```diff
@@ -235,107 +237,14 @@ class PersistingServiceSpec
 
       persistingService.persistTeamRepoMapping.futureValue
 
-      verify(dataSource).mapTeam(eqTo(ghTeamA), any(), any())
+      verify(dataSource).mapTeam(eqTo(ghTeamA), any())
 
       val mergedRepositories = teamARepositoriesInDataSource1.repositories.sortBy(_.name)
       verify(persistingService.persister)
         .update(teamARepositoriesInDataSource1.copy(repositories = mergedRepositories))
       verify(persistingService.persister)
         .update(unknownTeamRepositories)
     }
-
-    "process teams in the correct order so that the latest updated teams are processed last and teams that have not been processed are first" in {
```

Comment:
I don't think this feature really pays its way anymore. The cache is rebuilt in full on each run, so there doesn't seem to be a need to prioritise by the longest updated timestamp.